modules/k8s-demo-app/services/nginx.yaml

```diff
@@ -46,16 +46,16 @@ data:
         proxy_buffering off;
 
         location / {
-          proxy_pass http://127.0.0.1:3000;
+          proxy_pass http://frontend:3000;
```

Comment:
Needed this on deployments with consul-dataplane since transparent proxy is now enabled.